api/admin/controller/library_settings.py

```diff
@@ -73,8 +73,6 @@ def process_get(self):
         return dict(libraries=libraries, settings=Configuration.LIBRARY_SETTINGS)
 
     def process_post(self, validators_by_type=None):
-        self.require_system_admin()
-
```

Comment:
Should you call require_library_manager instead, or is this taken care of by the @requires_admin that guards the route?

Reply:
No, librarians can edit libraries too now, not just system admins and library managers.  @requires_admin should be plenty.